### PR TITLE
Create contact form

### PR DESCRIPTION
**Pull request by ayebaishmo**
In this branch the following was done:
Created a contact form with HTML
 Designed the contact form with CSS
Fixed all the lint errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,39 +184,45 @@ <h4 class="dark"><span class="figure-3"></span>Frameworks</h4>
...
+                    <ul>
+                      <li>
+                        <input class="inpt" type="text" placeholder="Full name" id="name" maxlength="30" name="name" required>
+                      </li>
+                      <li>
+                        <input class="inpt" type="email" placeholder="Email address" id="email" name="email" required>
+                      </li>
+                      <li>
+                        <textarea class="txt-a" name="message" id="message" placeholder="Enter text here" maxlength="500" required>
+                        </textarea>
```
Comment (DeliceLydia) [inline index.html:202]: - [ ] Check why this form allows sending an empty text area when i removed that class 
`  <textarea
                      class=""
                      id="message"
                      name="message"
                      maxlength="500"
                      placeholder="Enter text here"
                      required
                    ></textarea>`
 it worked perfectly. please validate it!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @ayebaishmo ✋, 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ayebaishmo) [reply index.html:202]: Hey It does not allow submission. Just press back space and remove the space.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Geoelycom — COMMENTED** [review]
Good job but you should limit the use of divs and use more semantic tags

**Review by eri8-9 — COMMENTED** [review]
- Good job 💪🏼
- Keep working on your ability to choose a better syntax.

**Review by eri8-9 — COMMENTED** [review]
- I think you already did your changes so great job so far

**Pull request merged**